decidim-proposals/app/views/decidim/proposals/proposal_votes/update_buttons_and_counters.js.erb

```diff
@@ -16,7 +16,7 @@
 
 <% if vote_limit_enabled? %>
   (function() {
-    var $remainingVotesCount = $('div[data-remaining-votes-count="true"]');
+    var $remainingVotesCount = $('[data-remaining-votes-count]');
```

Comment:
As far as I understand, this isn't changing anything, as both of these selectors would work with the HTML https://github.com/decidim/decidim/blob/efaf20097c17dbc6247016d127d36163ec865f8d/decidim-proposals/app/views/decidim/proposals/proposals/_remaining_votes_count.html.erb#L1